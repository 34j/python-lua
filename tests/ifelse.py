a = 45

if a > 5:
    print("a > 5")
    if a >= 45:
        print("a >= 45")
    else:
        print("a < 45")
elif a < 5:
    print("a < 5")
else:
    print("a == 5")

if a == 45:
    print("a == 45")
