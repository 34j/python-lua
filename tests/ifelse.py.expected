a > 5
a >= 45
a == 45
